07-react-1/classworks/Emelyanov_Egor/src/store/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from "redux";
+
+import inputValue from "./inputValue";
+import subredditData from "./subredditData";
+
+export default combineReducers({
+  inputValue,
```

Comment:
i doubt you should create a reducer for one string. better name it `input` and it will be the object { value: '' }